packages/react-keytips/src/components/Keytips/useKeytips.tsx

```diff
@@ -165,10 +165,10 @@ export const useKeytips_unstable = (props: KeytipsProps): KeytipsState => {
     (ev: KeyboardEvent, node: KeytipTreeNode) => {
       tree.currentKeytip.current = node;
 
-      if (node.target) {
+      if (node.target && !node.target.hasAttribute('disabled')) {
```

Comment:
For components that are neither buttons nor inputs, `aria-disabled="true"` is used instead of the `disabled` attribute. It's important to review if there are additional patterns to be aware of.

<img width="1460" alt="Screenshot 2024-11-01 at 14 26 49" src="https://github.com/user-attachments/assets/c5e3da67-8b54-469c-9e3b-9abd874411df">